app/templates/create.html

```diff
@@ -0,0 +1,63 @@
+{% extends "base.html" %}
+
+{%block content%}
+<div class="container-fluid  min-vh-100 bg-light">
+  <h1 class="my-3 text-center">Create a Challenge</h1>
+  
+  <form class="container my-1 rounded bg-white border border-primary" id="createForm" data-link="{{ url_for('create') }}">
+    {{ form.hidden_tag() }}
+
+    <div class="row my-3 gy-2">
+      {{ form.title.label(class="col-form-label col-2") }}
+      <div class="col-10">
```

Comment:
The display breaks a bit on mobile - use col-sm-10 instead? 
![image](https://github.com/RedBlueCarrots/WikiPath/assets/144818607/94fdc170-a4ba-41bb-8ec7-468912dc07b5)

Reply:
I've changed all of the col-10 to col-sm-10 and I believe this has fixed the problem.